blog/arquillian-governor-1.0.4.Final.textile

```diff
@@ -0,0 +1,63 @@
+---
+date: 2016-09-20
+author: dipak-pawar
+---
+
+We are proud to announce new features in this release for Arquillian Governor:
+
+h3. New Features
+
+- <i class="icon-star"></i> Arquillian Recorder integration with Arquillian Governor:=
+
+p. In this release we are providing recorder integration with Jira, GitHub & Redmine Governor.
+
+p. Arquillian Recorder project brings neat reports of your Arquillian tests. Now we are providing an integration to the project so that generated report contains information about JIRA, GitHub and Redmine properties such as Issue URL, Detector & it's strategy used for environment, force value.
```

Comment:
@dipak-pawar @lordofthejars this paragraph is a bit blurry from the end-user perspective I believe. Is it about recorder or governor? What are these values used for?

> properties such as Issue URL, Detector & it's strategy used for environment, force value.

Can we rephrase it and explain what does it really stand for and how is it used?